bedrock/firefox/templates/firefox/enterprise/index.html

```diff
@@ -0,0 +1,50 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/wait-face/base.html" %}
```

Comment:
This is not going to work, because you're going to also inherit this line, and load a lang file that has nothing to do with this page
https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/new/wait-face/base.html#L5